image/stub/stub.go

```diff
@@ -25,6 +25,11 @@ type Stub struct {
 	t image.Type
 }
 
+// Next returns the next extent.
+func (img *Stub) Next(start, length int64) (image.Extent, error) {
+	return image.Extent{Start: start, Length: length, Allocated: true}, img.Readable()
```

Comment:
Need to fail out of bounds requests like qcow2 version.

Reply:
Fixed to always fail since we don't know the image size and the image is not readable.